include/directx/d3d12.h

```diff
@@ -466,8 +466,13 @@ typedef interface ID3D12GraphicsCommandList7 ID3D12GraphicsCommandList7;
 
 
 /* header files for imported files */
+#if defined(NTDDI_WIN10_FE) || defined(__MINGW32__)

```

Comment:
These files are generated and should not be modified directly. Instead the .idl files need to be modified. However, there is no way to encode this in the .idl files - attempting to wrap the `import` statements in `cpp_quote` to insert the `#if` lines produces:

```c++
/* header files for imported files */
#include "oaidl.h"
#include "ocidl.h"
#include "dxgicommon.h"
#include "dxgiformat.h"
#include "d3dcommon.h"

#ifdef __cplusplus
extern "C"{
#endif 


/* interface __MIDL_itf_d3d12_0000_0000 */
/* [local] */ 

#if defined(NTDDI_WIN10_FE) || defined(__MINGW32__)
#else
#include "OAIdl.h"
#include "OCIdl.h"
#endif
```

Given that, and the fact that these headers were actually changed in the Windows SDK in 2020, I think I'm okay with your originally proposed change of simply doing the rename. Please also include the change to the .idl files as part of that change. The only folks who would then be broken are people using `DirectX-Headers` with an old Windows SDK on a case-sensitive filesystem, and that case should be rare enough that I don't think it's a concern for now.

Reply:
There is no idl file to rename.

Reply:
Sorry I'm not saying to rename the .idl files, I'm saying to change the contents of them so that they `import "oaidl.idl;"` instead of `import "OAIdl.idl;"` (and same for ocidl). Essentially, a revert of https://github.com/microsoft/DirectX-Headers/commit/5b5b57aee33e8e38eff0242221893a43ac6ea90d.

Reply:
done